apps/kyb-app/src/components/organisms/UIRenderer/elements/JSONForm/components/DocumentField/DocumentField.tsx

```diff
@@ -106,22 +103,23 @@ export const DocumentField = (
         setFieldError(null);
       } catch (err) {
         if (err instanceof HTTPError) {
-          const jsonError = await err.response.json();
-
+          const response = (await err.response.json()) as AnyObject;
           setFieldError({
             fieldId: document.id,
-            message: jsonError.message,
+            message: response.message as string,
             type: 'warning',
           });
+          return;
         }
-      }
 
-      toggleElementLoading();
+        throw err;
```

Comment:
It will throw the user from his page to the Error Boundary fallback

Reply:
ErrorBoundary catches rendering exceptions and not callbacks.